xml/templates/message_templates/contribution_invoice_receipt_html.tpl

```diff
@@ -105,246 +92,228 @@
               <tr>
-                <td colspan = "3"></td>
-                <td style = "padding-left:20px;text-align:right;"><font size = "1">{ts}Sub Total{/ts}</font></td>
-                <td style = "padding-left:34px;text-align:right;"><font size = "1"> {$subTotal|crmMoney:$currency}</font></td>
+                <td colspan="3"></td>
+                <td style="text-align:right;"><font size="1">{ts}Sub Total{/ts}</font></td>
+                <td style="text-align:right;"><font size="1">{$subTotal|crmMoney:$currency}</font></td>
               </tr>
-              {foreach from = $dataArray item = value key = priceset}
+              {foreach from=$dataArray item=value key=priceset}
                 <tr>
-                  <td colspan = "3"></td>
+                  <td colspan="3"></td>
                     {if $priceset}
-                      <td style = "padding-left:20px;text-align:right;"><font size = "1"> {ts 1=$taxTerm 2=$priceset}TOTAL %1 %2%{/ts}</font></td>
-                      <td style = "padding-left:34px;text-align:right"><font size = "1" align = "right">{$value|crmMoney:$currency}</font> </td>
+                      <td style="text-align:right;white-space: nowrap"><font size="1">{ts 1=$taxTerm 2=$priceset}TOTAL %1 %2%{/ts}</font></td>
+                      <td style="text-align:right"><font size="1" align="right">{$value|crmMoney:$currency}</font> </td>
                     {elseif $priceset == 0}
-                      <td style = "padding-left:20px;text-align:right;"><font size = "1">{ts 1=$taxTerm}TOTAL NO %1{/ts}</font></td>
-                      <td style = "padding-left:34px;text-align:right"><font size = "1" align = "right">{$value|crmMoney:$currency}</font> </td>
+                      <td style="text-align:right;white-space: nowrap"><font size="1">{ts 1=$taxTerm}TOTAL %1{/ts}</font></td>
+                      <td style="text-align:right"><font size="1" align="right">{$value|crmMoney:$currency}</font> </td>
                 </tr>
               {/if}
               {/foreach}
               <tr>
-                <td colspan = "3"></td>
-                <td colspan = "2"><hr></hr></td>
-              </tr>
-              <tr>
-                <td colspan = "3"></td>
-                <td style = "padding-left:20px;text-align:right;"><b><font size = "1">{ts 1=$defaultCurrency}TOTAL %1{/ts}</font></b></td>
-                <td style = "padding-left:34px;text-align:right;"><font size = "1">{$amount|crmMoney:$currency}</font></td>
-                <td style = "padding-left:34px;"><font size = "1" align = "right"></font></td>
+                <td colspan="3"></td>
+                <td style="text-align:right;white-space: nowrap"><b><font size="1">{ts 1=$defaultCurrency}TOTAL %1{/ts}</font></b></td>
+                <td style="text-align:right;"><font size="1">{$amount|crmMoney:$currency}</font></td>
               </tr>
-              {if $is_pay_later == 0}
+             {if $amountDue != 0}
```

Comment:
This caused a regression - see https://github.com/civicrm/civicrm-core/pull/18602